src/azure-cli/azure/cli/command_modules/keyvault/_transformers.py

```diff
@@ -474,3 +474,13 @@ def transform_certificate_issuer_admin_list(result, **command_args):
         } for contact in admin_contacts]
         return ret
     return result
+
+
+def transform_security_domain_output(result, **command_args):
+    if not result or isinstance(result, dict):
+        return result
+    ret = {
+        'status': getattr(result, 'status', None),
+        'statusDetails': getattr(result, 'status_details', None)
+    }
```

Comment:
Because SDK use a strange [ModelBase](https://github.com/mccoyp/azure-sdk-for-python/blob/000f7af897937c8146dea5da28cd74dd4ef0f16e/sdk/keyvault/azure-keyvault-securitydomain/azure/keyvault/securitydomain/_model_base.py#L493) which can't be serialized by knack